app/views/workload-points.html

```diff
@@ -471,6 +475,18 @@
               </table>
             </div>
           </div>
+          {% endif %}
+          {% if wp.isT2A %}
+          <input class="form-control" id="sdr" name="sdr" type="hidden" value="{{ wp.sdr }}">
```

Comment:
These values aren't needed for T2A? Do they need to be included here? What happens if they are removed - will the form still submit but with `null`s?

Reply:
If these values aren't passed in the the workload validator fails the validation and the object isn't created. We could have a separate workload object for T2A. Given that both the standard and t2a are using the same table its make sense to have a single workload point object. Another possibility would not to validate these values for T2A points based on the isT2A flag.